Makefile

```diff
@@ -7,15 +7,23 @@ SBT:=./sbt
 all: snappy
 
 SNAPPY_OUT:=$(TARGET)/$(snappy)-$(os_arch)
-SNAPPY_ARCHIVE:=$(TARGET)/snappy-$(VERSION).tar.gz 
+SNAPPY_ARCHIVE:=$(TARGET)/snappy-$(VERSION).tar.gz
 SNAPPY_CC:=snappy-sinksource.cc snappy-stubs-internal.cc snappy.cc
 SNAPPY_SRC_DIR:=$(TARGET)/snappy-$(VERSION)
 SNAPPY_SRC:=$(addprefix $(SNAPPY_SRC_DIR)/,$(SNAPPY_CC))
-SNAPPY_OBJ:=$(addprefix $(SNAPPY_OUT)/,$(patsubst %.cc,%.o,$(SNAPPY_CC)) SnappyNative.o)
-
 SNAPPY_UNPACKED:=$(TARGET)/snappy-extracted.log
 SNAPPY_GIT_UNPACKED:=$(TARGET)/snappy-git-extracted.log
 
+ifdef ENABLE_BITSHUFFLE
+  # TODO: Upgrade to a stable release
+  BITSHUFFLE_VERSION:=55f9b4caec73fa21d13947cacea1295926781440
```

Comment:
Is this a stable version? Do you know how often this library needs update? Collecting native libraries for snappy-java is a hard task, so I would like to avoid rebuilding native libraries as much as possible. 

It looks like you are still working on updating bitshuffle repository. To merge it to the master, we need to make this bitshuffle library stable enough.